modules/dnn/src/dnn_common.hpp

```diff
@@ -15,6 +15,17 @@ void initializeLayerFactory();
 
 namespace detail {
 
+class NotImplemented : public Layer
+{
+public:
+    static Ptr<Layer> create(const LayerParams &params);
+
+    static void Register();
+    static void unRegister();
```

Comment:
Register -> register

https://github.com/opencv/opencv/wiki/Coding_Style_Guide#naming-conventions

Reply:
This is a reserved keyword.